lib/Controller/LockController.php

```diff
@@ -94,7 +93,7 @@ public function locking(string $fileId): DataResponse {
 			$user = $this->userSession->getUser();
 			$file = $this->fileService->getFileFromId($user->getUID(), (int)$fileId);
 
-			$lock = $this->lockService->lockFile($file, $user);
+			$lock = $this->lockService->lockFileAsUser($file, $user);
```

Comment:
I haven't test does `lockFileAsUser()` exist in `LockService` ?

Reply:
Whoops, good catch, seems I failed to commit only the relevant parts here, this is from #54